### DISCUSSION TITLE
Post process failure type indication

### DISCUSSION THREAD

**Discussion by nydave69** [pymedusa/Medusa#9144] [Ideas & Feature requests]
A text file matching the name of the file indicating type of failure located with the failing file.

The text file could contain information related to why it failed post processing.  Very helpful if your looking at files that failed a day ago.  Also the extension 'error' could be added to the file to give better visibility.  And then on sequential post-processing passes it could use the error file as a flag that is has been processed, skip it.  Deleting the error file once the error was fixed (ex. bad file name) would once again let post-processing happen normally to that file or group of files if it's in the sub folder..

**Comment by p0psicles** [comment]
I think this is a bit of a hacky solution. What's the real issues you facing. Maybe we can help you out with the tools already available?

**Reply by nydave69** [reply]
I check and find unprocessed files from over night.

**Reply by p0psicles** [reply]
Can you go into a bit more detail. And talk about the alternative solutions you've considered?

**Reply by nydave69** [reply]
I use FileBot to process the file, Move it to were it goes. Have Medusa re-scan the files for that show.
The Medusa log is usually useless because from the time it was processed and I find them, is hours..
That's why it would be nice if the error was logged with the file.   that's why I suggest it. 
There are programs in other categories that do exactly what is suggested to make things easier for the user, to match the error to the file in question.

I'm stumped as to why you think it's a "hacky solution".
